### PR TITLE
Added DifferentFirstPageHeader and Page Number features

### PR DESCRIPTION
**Pull request by formatically**
This pull request features "DifferentFirstPageHeader" support.  Also, provides the ability to add page numbers to the headers. 

Closes https://github.com/dolanmiu/docx/issues/72

### CODE REVIEW

File: src/file/document/body/section-properties/header-reference/header-reference.ts [hunk 1]
```diff
@@ -2,12 +2,12 @@ import { XmlComponent } from "file/xml-components";
...
 export class HeaderReference extends XmlComponent {
-    constructor() {
+    constructor(order, ref_id) {
         super("w:headerReference");
         this.root.push(
             new HeaderReferenceAttributes({
-                type: "default",
-                id: `rId${3}`,
+                type: order,
+                id: `rId${ref_id}`,
```
Comment (dolanmiu) [inline src/file/document/body/section-properties/header-reference/header-reference.ts]: Could this be refId

Just personal preference

**Comment by dolanmiu** [comment]
Good work!

Once you fix those comments I mentioned, and fix the build, I'll merge :)

**Comment by formatically** [comment]
I think it's good to go! If you see anything I still need to change, just let me know.

**Comment by dolanmiu** [comment]
Merged, will release a new version soon!

**Comment by formatically** [comment]
Awesome sounds great!

**Pull request merged**